patches/aspnetcore/0010-Support-building-for-arm64-on-arm64-nix.patch

```diff
@@ -0,0 +1,85 @@
+From e2946a26c11be7f7f0c223721a5b14f58f2ea240 Mon Sep 17 00:00:00 2001
```

Comment:
This is https://github.com/aspnet/AspNetCore/pull/15354. **MERGED** into `master`.